browser/ui/sidebar/sidebar_controller.h

```diff
@@ -43,12 +43,23 @@ class SidebarController : public SidebarService::Observer,
   SidebarController(const SidebarController&) = delete;
   SidebarController& operator=(const SidebarController&) = delete;
 
+  // NOTE: Don't call this directly for panel item. Use ActivatePanelItem().
+  // This should be called as a result of SidePanelCoordinator's entry
+  // opening/closing event. If this method is called directly for activating
+  // panel, SidePanelCoordinator doesn't know about it.
+
   // |disposition| is only valid for shortcut type. If |disposition| is not
   // CURRENT_TAB, item at |index| is handled based on |disposition|.
   void ActivateItemAt(
       absl::optional<size_t> index,
       WindowOpenDisposition disposition = WindowOpenDisposition::CURRENT_TAB);
   void AddItemWithCurrentTab();
+
+  // Ask panel item [de]activation to SidePanelUI. If it's called w/o arg, panel
+  // will be closed.
+  void ActivatePanelItem(SidebarItem::BuiltInItemType panel_item =
```

Comment:
Can we remove the default arg or having another method passing kNone to this method? When I was reading a callsite without the argument, I was quite confused by that calling ActivatePanelItem() was doing the exactly opposite thing.

```
  void ActivatePanelItem(SidebarItem::BuiltInItemType panel_item);
  void DeactivateCurrentPanel() { ActivePanelItem(kNone); }
```